docs/new/posts/PostgreSQL/postgresql-17.5.1-release.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2025-05-28
+description: >
+  Percona Distribution for PostgreSQL 17.5.1 has been released on May 28, 2025.
+authors: [Andriciuc]
+categories:
+  - PostgreSQL
+tags:
+  - Percona Distribution for PostgreSQL
+---
+
+# Percona Distribution for PostgreSQL 17.5.1 has been released
+
+<!-- more -->
+
+[Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/17/index.html){:target="_blank"} 17.5.1 has been released on May 28, 2025.
+
+Try it out using the [Quickstart guide](https://docs.percona.com/postgresql/17/installing.html){:target="_blank"}. Check the [upgrade instructions](https://docs.percona.com/postgresql/17/major-upgrade.html){:target="_blank"}
+
+This release of Percona Distribution for PostgreSQL 17.5.1 completes the quarterly minor release cycle for PostgreSQL quarter 2. It provides the following enhancements:
```

Comment:
It cannot complete the quarter releases while it starts it:) 
Please remove this line